assets/dev/js/frontend/handlers/playing-cards.js

```diff
@@ -0,0 +1,232 @@
+import Base from 'elementor/assets/dev/js/frontend/handlers/base';
+
+export default class PlayingCards extends Base {
+	cards = [];
```

Comment:
I am just curious. Do we not need to use `const` or `let` here?

Reply:
@hein-obox No it's class properties.